indico/modules/events/surveys/controllers/management.py

```diff
@@ -164,6 +166,7 @@ class RHOpenSurvey(RHManageSurvey):
     def _process(self):
         if self.survey.state == SurveyState.finished:
             self.survey.end_dt = None
+            self.survey.start_notification_sent = False
```

Comment:
Wouldn't it make more sense to always set this to false? The state wouldn't be finished e.g. when you remove all questions